Readme test (#1)
t-h2o/inception: README.adoc

== Create a VM

> . launch my interactive menu using the *_create_vm.sh_* script (located in the "scripts" folder)

Reviewer: the informations are repeat two times.
1. once in this sentence.
2. a second time in the command below.
should we shoot one of the both ?

Author: Definitely, the "image" of the menu could directly go with 1.

Reviewer: What is `the "image" of the menu` ?

Author: Just the display of the menu, the different options

+
[,bash]
----
sh scripts/create_vm.sh
----
 
. the interactive menu looks like this :
+
[,bash]
----
main menu:
1: setup the vm
2: list vms
3: exit
4: start the vm
5: generate info-vm.txt
6: remove archlinux iso
7: delete the vm
----

. select the first option to begin the setup of a new VM and follow the steps

> 

Reviewer: AsciiDoc: [list continuation](https://docs.asciidoctor.org/asciidoc/latest/lists/continuation/#list-continuation)
```suggestion
+
```

Author: That's what I was looking for, thx!

>   you can skip each step if you know what you are doing

Reviewer: AsciiDoc: [admonitions](https://docs.asciidoctor.org/asciidoc/latest/blocks/admonitions/#admonition-types)

It's a builtin to mark text as note.

```suggestion
NOTE: you can skip each step if you know what you are doing
```

** `*_create the vm_*` creates the .vbox file which contains the settings needed by VirtualBox to create the VM
** `*_download_archlinux_iso_*` downloads and place an archlinux disk image (.iso) in the _my_vm_created_by_cli/_ folder
** `*_attach_archlinux_iso_*` will "plug" the .iso as you would do with a bootable device to allow the VM to boot using it
** `*_create_vdi_*` creates the _virtual disk image_ (.vdi) file where the VM is created
** `*_create_vm_sata_*` creates the sata drive for the VM
** `*_start_vm_*` will do what it should do
** The following error should appear, just hit _Change Network Settings_ and hit _OK_ (default options should be fine) +
image:images/error_net.png[,250]
image:images/net_ok.png[,250]
** boot up the VM +
image:images/boot_screen.png[,250]
image:images/VM_running.png[,250]

. It can be good idea at this point to setup an SSH connection

> .. J'ai oublie comment on fait

Reviewer: 1. `ip a`
   > get the IP address of the virtual machine

2. `passwd`
   > set a default password to the `root` user. Tt is a requirement to connect by `ssh` to your live session
   
3. `ssh root@{IP_ADDRESS}`

** `*_create_vdi_*` creates the _virtual disk image_ (.vdi) file where the VM is created
** `*_create_vm_sata_*` creates the sata drive for the VM
** `*_start_vm_*` will do what it should do
** The following error should appear, just hit _Change Network Settings_ and hit _OK_ (default options should be fine) +
image:images/error_net.png[,250]
image:images/net_ok.png[,250]
** boot up the VM +
image:images/boot_screen.png[,250]
image:images/VM_running.png[,250]

. It can be good idea, at this point, to setup an SSH connection :
.. Find the ip address of the machine. You can use the following command :
+
[,bash]
----
ip a
----
+
The `_a_` stands for _address_. This will display information about all network interfaces, your ip address is somewhere here.
.. Set your root user password with :
+
[,bash]
----
passwd
----
+
the default password in the VM will be _1234_

.. Create the SSH connection using the root username and ip address :
+
[,bash]
----
ssh root@{IP_ADRESS}
----

. Once in the vm, launch the following command to setup the environment
+
[,bash]
----
sh -c "$(curl -fsSL https://raw.githubusercontent.com/t-h2o/inception/main/scripts/live_install_arch.sh)"
----

> . Go back to the interactive menu and select `*_6: remove archlinux iso_*`

Reviewer: before:

1. shutdown the VM

after, for using the VM:

1. log in with your username
2. `ip a`
3. `ssh $(whoami)@${IP_ADDRESS}`

+

> This will "unplug" the iso to avoid the previous installation process when starting up the vm

Reviewer: ```suggestion
This will "unplug" the iso to avoid to boot on the live Linux session
```

image:images/boot_screen.png[,250]
image:images/VM_running.png[,250]

. It can be good idea, at this point, to setup an SSH connection :
.. Find the ip address of the machine. You can use the following command :
+
[,bash]
----
ip a
----
+
The `_a_` stands for _address_. This will display information about all network interfaces, your ip address is somewhere here.
.. Set your root user password with :
+
[,bash]
----
passwd
----
+

> the default password in the VM will be _1234_

Reviewer: the default password information can be say later.
When you explain how use the VM after the Linux installation
